### DISCUSSION TITLE
Increasing Generation - PV or Slack Bus without Generator

### DISCUSSION THREAD

**Discussion by tyler477** [CURENT/andes#541] [Modeling]
Hello, I am new to development in Andes. 

I know i can change the load in PQ buses, but how would I go about changing the power provided from a PV bus without employing a generator element (e.g., as a step response)?  To clarify, I want to start with basics as load increase by .1 pu and generator increase by .1 pu to compensate.

**Comment by cuihantao** [comment]
Hello, to change the active power output from a PV generator, you can update the `p` field of the `PV`.

`ss.PV.p.v[index] = desired_value` should do. You can also use the `ss.PV.set()` interface.

Doing so without dynamic components, however, has no difference from running two separate power flows. You will simply see a jump in the algebraic solutions without any dynamics.
